src/frontend/src/lib/components/networks/MainnetNetwork.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+	import type { Network } from '$lib/types/network';
+	import NetworkComponent from '$lib/components/networks/Network.svelte';
+	import { enabledMainnetTokensUsdBalancesPerNetwork } from '$lib/derived/tokens.derived';
+
+	export let network: Network;
+
+	let totalUsd: number;
+	$: totalUsd = $enabledMainnetTokensUsdBalancesPerNetwork[network.id];
+</script>
+
+<NetworkComponent {network} {totalUsd} on:icSelected />
```

Comment:
I would rename it `usdBalance` just to have the same convention

Reply:
Good point, renamed